$NetBSD: patch-man_spectrumctl.8,v 1.1.1.1 2011/02/07 06:31:22 schnoebe Exp $

Change out the /etc/spectrum path for easy substitution
Change out the spectrum username for easy substitution.

--- man/spectrumctl.8.orig	2010-08-12 15:18:20.000000000 +0000
+++ man/spectrumctl.8
@@ -23,7 +23,7 @@ with more complex administration tasks. 
 explanation.
 .SH ACTIONS
 .sp
-By default, spectrumctl acts on all transports defined in \fI/etc/spectrum/\fR.
+By default, spectrumctl acts on all transports defined in \fI@ETC_SPECTRUM@\fR.
 You can use a different directory with the \fI--config-dir\fR option. If you
 just want to act on a single file, you can use the \fI--config\fR option. Unless
 you use \fI--config\fR, spectrumctl will silently ignore files where the
@@ -101,7 +101,7 @@ Only act on transport configured in FILE
 .RE
 \fB\-d\fR \fIDIR\fR, \fB\-\-config\-dir\fR=\fIDIR\fR
 .RS 4
-Act on all transports configured in DIR (default: \fI/etc/spectrum\fR)
+Act on all transports configured in DIR (default: \fI@ETC_SPECTRUM@\fR)
 .sp
 .RE
 \fB\-q\fR, \fB\-\-quiet\fR
@@ -146,9 +146,9 @@ Options for action "\fBstart\fR":
 .RS 4
 \fB\-\-su\fR=\fISU\fR
 .RS 4
-Start spectrum as this user (default: \fIspectrum\fR). Overrides the SPECTRUM_USER environment variable.
+Start spectrum as this user (default: \fI@SPECTRUM_USER@\fR). Overrides the SPECTRUM_USER environment variable.
 If neither \fI\-\-su\fR nor the environment variable \fBSPECTRUM_USER\fR is specified, the user defaults to 
-\fIspectrum\fR.
+\fI@SPECTRUM_USER@\fR.
 .RE
 .sp
 \fB\-\-no-daemon\fR
@@ -168,7 +168,7 @@ multiple administration tasks at once. Y
 commands only available within the shell. 
 .sp
 The prompt of the spectrumctl shell represents the transports that commands will
-act upon. If the shell acts upon all config-files defined in \fI/etc/spectrum\fR,
+act upon. If the shell acts upon all config-files defined in \fI@ETC_SPECTRUM@\fR,
 the prompt will show \fB<all transports>\fR and the JID of the current transport
 otherwise.
 .sp
@@ -198,7 +198,7 @@ Path where the spectrum binary is locate
 The user with which spectrum is started. Overridden by the --su command line
 option. 
 If neither --su nor SPECTRUM_USER is specified, the user defaults to 
-\fIspectrum\fR.
+\fI@SPECTRUM_USER@\fR.
 .RE
 .SH AUTHORS
 Copyright \(co 2009\-2010 by Spectrum engineers:
